tests/data/test_dataloader.py

```diff
@@ -437,7 +437,7 @@ def test_finetuning_dataloader_safe_load(
         'dataset': {
             'hf_name': hf_name,
             'split': 'train',
-            'max_seq_len': 8,
+            'max_seq_len': 100,
```

Comment:
jw, any particular reason to make this change?

Reply:
the dataset was empty bc the max_seq_len was too short and causing the new error to be raised :)